netutils/config/parser.py

```diff
@@ -949,3 +949,63 @@ def _build_nested_config(self, line):
                 self.indent_level = spaces
 
             self._update_config_lines(line)
+
+
+class NokiaConfigParser(BaseSpaceConfigParser):
+    """Nokia SrOS config parser."""
+
+    comment_chars = ["# ", "#--------------------------------------------------"]
```

Comment:
This confuses me a bit, I would think just having `#` would catch both of these?